resources/netregexes.ts

```diff
@@ -102,9 +120,12 @@ export default class NetRegexes {
     });
   }
 
-  // matches: https://github.com/quisquous/cactbot/blob/main/docs/LogGuide.md#15-networkability
-  // matches: https://github.com/quisquous/cactbot/blob/main/docs/LogGuide.md#16-networkaoeability
-  static abilityFull(params) {
+  const abilityFullParams = ['sourceId', 'source', 'id', 'ability', 'targetId', 'target', 'flags', 'damage', 'targetCurrentHp', 'targetMaxHp', 'x', 'y', 'z', 'heading'] as const;
+  /**
+   * matches: https://github.com/quisquous/cactbot/blob/main/docs/LogGuide.md#15-networkability
+   * matches: https://github.com/quisquous/cactbot/blob/main/docs/LogGuide.md#16-networkaoeability
```

Comment:
What is the comment style change here for?

Reply:
JSDoc style, so most editor with JSDoc extension would display these text when hover it or type it. (VSCode support this as built-in feature though.)

Reply:
Just like:
![image](https://user-images.githubusercontent.com/19927330/107865946-b24dc600-6ea6-11eb-989a-0443158b4781.png)